docs/samples/python/presidio_notebook.ipynb

```diff
@@ -30,7 +34,7 @@
    "source": [
     "from presidio_analyzer import AnalyzerEngine, PatternRecognizer\n",
     "from presidio_anonymizer import AnonymizerEngine\n",
-    "from presidio_anonymizer.entities import AnonymizerRequest"
```

Comment:
imports can be done mid-code, and in this case moved to the bottom source clause , so that it will be clearer how to use each service in separate.

Reply:
they can, but the notebook is more organized this way.